### DISCUSSION TITLE
add a class to ng-dropdown-panel when appendTo is activate

### DISCUSSION THREAD

**Discussion by pboutin44** [ng-select/ng-select#2188] [Ideas]
Hello, I am a front dev and I work on a project that use ng-select with appendTo='body'.
I would like to focus ng-dropdown-panel with css to add style so I propose the PR: #2187.

It is my first PR on github ☺️, if you have time to tell me what you think about my proposal I will be grateful.

Thanks for reading me

Pierre

**Comment by pavankjadda** [comment]
This has been added in recent version
